### DISCUSSION TITLE
Incompatible V6 official packages

### DISCUSSION THREAD

**Discussion by kaypee90** [adonisjs/core#4409] [General]
In the migration guide, https://v6-migration.adonisjs.com/guides/introduction, there were several packages that were not compatible but were going to be available in a few weeks.
Are there any updates and is there a place to track progress on this?

Also Health checks package was removed to its own package. Where do I find it to install?

**Comment by muco-rolle** [comment]
This article(https://adonisjs.com/blog/) doesn't present a specific schedule, but it sheds light on potential developments
